uber/templates/registration/history.html

```diff
@@ -67,3 +67,25 @@
 {% endfor %}
 
 {% endblock %}
+{% block scripts %}
+    <script>
+        var resend = function(email_id){
+            $.ajax(
+                    {
+                        method: 'POST',
+                        url:'resend_email',
+                        data:{
+                            csrf_token: csrf_token,
+                            id: email_id
+                        },
+                        success: function(response){
```

Comment:
I am not hip to the AJAX, so I am not sure how this function knows whether the function it calls it successful or not when you return a string either way. Also, does this mean that `message` is not ever actually shown to the administrator?